lib/css/modules/_userTagger.scss

```diff
@@ -1,7 +1,3 @@
-.comment .tagline {
-	display: inline;
```

Comment:
I thought this might have been the fix for comments getting pushed below the sidebar, but it looks like it's been here forever: https://github.com/honestbleeps/Reddit-Enhancement-Suite/blame/5bef0d2a2edc6fcf06fe760ffa2cb27d52d3eb94/lib/modules/userTagger.js#L51

So I don't see a reason to keep it either.